### PR TITLE
Removing DHCPCIDR docs

### PR DESCRIPTION
**Pull request by stephenrlouie**

### CODE REVIEW

File: docs/current/README.md [hunk 1]
```diff
@@ -53,49 +53,48 @@ Vaquero can run in multiple modes: `server`, `agent`, and `standalone`. This con
...
- Port: 8080
- BaseDir: "/tmp/vaquero/files"
- Scheme: http
-DHCPMode: server
-DHCPCIDR: "127.0.0.1/16"
+  Addr: "127.0.0.1"
+  Port: 8080
+  BaseDir: "/tmp/vaquero/files"
+  Scheme: http
+DHCP:
```
Comment (mikeynap) [inline docs/current/README.md]: Not exactly. Change this to JUST be 
- DHCPMode: 
  (i.e. revert to the old version, but remove DHCPCIDR). See https://github.com/CiscoCloud/vaquero/issues/333 for further clarification.

File: docs/current/README.md [hunk 2]
```diff
@@ -53,49 +53,48 @@ Vaquero can run in multiple modes: `server`, `agent`, and `standalone`. This con
...
 ```
 ************************************************************
 
 #### Explanation of config fields:
 - ServerApi: The user api for the server. Currently not implemented.
 - AgentApi: The vaquero-agent http server used to listen for vaquero server commands.
 - AssetServer: The asset server for vaquero agent used by each booting machine to get unattended scripts and kernels.
-- DHCPMode: One of two modes, `proxy` or `server`. Leaving the DHCP fields empty will disable all DHCP vaquero functionality.
-- DHCPCIDR: The CIDR managed by DHCP.
+- DHCP: One of two modes, `proxy` or `server`. DHCP endpoint address. Leaving the DHCP fields empty will disable all DHCP vaquero functionality.
```
Comment (mikeynap) [inline docs/current/README.md]: DHCPMode: `proxy`, `server` or `none` (maybe off?).  Leaving the DHCPMode field empty will disable all DHCP vaquero functionality.  

DHCPMode: Proxy enables ProxyDHCP. ProxyDHCP works with an existing DHCP Server to provide PXEBoot functionality, while leaving the managing and assigning of IP addresses to the other DHCP Server. ONLY enable this if you already have a DHCP server with entries for all the hosts in your Data Model.

DHCPMode: server runs vaquero as a DHCP server.  Vaquero does not manage free address pools or leases; it simply assigns based of the static configuration defined in the data model.

Comment (mikeynap) [reply docs/current/README.md]: *Something along those lines.

**Review by mikeynap — CHANGES_REQUESTED** [review]
Proposed Documentation Changes.

**Comment by stephenrlouie** [comment]
@mikeynap fixed

**Pull request merged**